cmd/kubelet/app/server.go

```diff
@@ -488,6 +446,35 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 		}
 	}
 
+	// Register current configuration with /configz endpoint
```

Comment:
Hey no worries. I'm glad that you're trying to work on things :). We seriously appreciate help any way you can give it.